scripts/build-all-in-one-image.sh

```diff
@@ -45,17 +45,96 @@ upload_to_docker() {
   fi
 }
 
-make build-all-in-one GOOS=linux GOARCH=$GOARCH
+upload_multiarch_to_docker() {
```

Comment:
Why is there so much code just for all-in-one? I assume it would be the same for other Docker images, so let's keep a single copy.

Reply:
Yes, you're right. I've extracted the duplicate code from `scripts/build-all-in-one-image.sh` and `scripts/build-upload-docker-images.sh` and put them in newly created scripts.